src/Versity.Users/External/Presentation/Controllers/UsersController.cs

```diff
@@ -0,0 +1,21 @@
+﻿using MediatR;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Presentation.Abstractions;
+
+namespace Presentation.Controllers;
+
+[Route("api/[controller]/")]
+public sealed class UsersController : ApiController
+{
+    public UsersController(ISender sender) : base(sender)
+    {
+    }
+
+    [Authorize(Roles = "Admin")]
+    [HttpGet]
+    public async Task<IActionResult> GetSomething()
```

Comment:
?
Will it be implemented?

Reply:
This is fixed with the following commit -> [Endpoints](https://github.com/jejikeh/versity/pull/3/commits/56cb777a5320bd0c8b836fce35d34a4afc02e27f#diff-9c2146eec9b7130c9342c4cf240125d490dd1168e2a328e3ed418043af96db97)